prestodb/spark-master/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM prestodb/spark-base:latest
```

Comment:
please make the tag (latest) as a build arg, so that we can specify it in the building process.

The initial base image will have the tag `3.5.1`.

Reply:
I added this as an ARG variable